config/install/media_alias_display.settings.yml

```diff
@@ -0,0 +1,4 @@
+_core:
```

Comment:
This configuration cannot reside here and you don't need it, as it is defined by another module and cannot be imported.